lib/services/auth-service.js

```diff
@@ -86,38 +95,17 @@ function authServiceFactory(
                     { jwtFromRequest: ExtractJwt.fromHeader('x-auth-token') }),
             self.jwtStrategyRedfish.bind(self)));
         passport.use(new BasicStrategy(self.localStrategyAuth.bind(self)));
+        passport.use(new AnonStrategy());
 
         return passportMidware;
     };
 
-    AuthService.prototype.loadFromConf = function (){
-        var username = configuration.get('authUsername', 'admin');
-        var passwordHash = configuration.get('authPasswordHash',
-            'KcBN9YobNV0wdux8h0fKNqi4uoKCgGl/j8c6YGlG7iA0PB3P9ojbmANGhD' +
-            'lcSBE0iOTIsYsGbtSsbqP4wvsVcw==');
-        var salt = configuration.get('authPasswordSalt',
-            'zlxkgxjvcFwm0M8sWaGojh25qNYO8tuNWUMN4xKPH93PidwkCAvaX2JItL' +
-            'A3p7BSCWIzkw4GwWuezoMvKf3UXg==');
-        var secret = configuration.get('authTokenSecret',
-            'RackHDRocks!');
-
-        assert.string(username, 'Invalid username in configure file');
-        assert.string(passwordHash, 'Invalid password hash in configure file');
-        assert.string(salt, 'Invalid crypto salt in configure file');
+    AuthService.prototype.loadFromConf = function () {
+        var secret = configuration.get('authTokenSecret', 'RackHDRocks!');
```

Comment:
Could you add some detail into http://rackhd.readthedocs.org/en/latest/packaging_and_deployment.html#configuration-parameters with this value, and it's default